src/pages/DataAccessRequestApplication.js

```diff
@@ -545,38 +664,81 @@ class DataAccessRequestApplication extends Component {
         prev.formData.datasetIds = datasetIds;
         prev.formData.ontologies = ontologies;
         return prev;
-      }, () => this.savePartial());
+      }, () => this.saveDarDraft());
     } else {
       this.setShowDialogSave(false);
     }
   };
 
-  savePartial = () => {
-    let formData = this.state.formData;
+
+  //NOTE: work on this, expect similar issue with submit?
```

Comment:
nit: Can you expand on this? What more work do we need to do?

Reply:
Another old comment, will remove